cypress/e2e/management-list-item-actions.feature

```diff
@@ -29,6 +33,17 @@ Feature: Dashboard filtering
       | Joker       | Publish,Request changes,Unpublish             | Update live version,Remove |
 
 
+
+  Scenario Outline: Show single radio button option for certain scenarios
```

Comment:
The scenario should be slightly more descriptive. `Show only unpublish when currently published or isAnnualReview` rather than just "certain scenarios" (i.e. what are the certain scenarios?). 

This will only check if "Unpublish" is present, but it won't check if the other radio buttons are missing, so the scenario name doesn't match up to what is actually being tested.

Reply:
All good points. I'll amend the test accordingly